ibis/tests/all/test_geospatial.py

```diff
@@ -0,0 +1,112 @@
+""" Tests for geo spatial data types"""
+import ibis
+from ibis.tests.backends import MapD
+import ibis.tests.util as tu
+from numpy import testing
+
+
+@tu.skipifnot_backend(MapD)
+def test_geo_op(backend, geo):
```

Comment:
Could You decompose and/or `pytest.parametrize` it to multiple test functions?

Reply:
sounds good @kszucs . thanks for the feedback. I am working on that.